cmd/loom/sendtx.go

```diff
@@ -53,26 +53,35 @@ func newMigrationCommand() *cobra.Command {
 			if callerChainID == "" {
 				callerChainID = cli.TxFlags.ChainID
 			}
-			return migrationTx(Id, cli.TxFlags.PrivFile, cli.TxFlags.Algo, callerChainID)
+			return migrationTx(Id, cli.TxFlags.PrivFile, cli.TxFlags.Algo, callerChainID, cli.TxFlags.CfgFilePath)
 		},
 	}
 	migrationCmd.Flags().Uint32Var(&Id, "id", 0, "migration ID")
 	migrationCmd.Flags().StringVarP(&cli.TxFlags.PrivFile, "key", "k", "", "private key file")
+	migrationCmd.Flags().StringVarP(&cli.TxFlags.CfgFilePath, "config", "p", "", "config file")
```

Comment:
Instead of `cli.TxFlags.CfgFilePath` use a local var named `inputFilePath`, just like we do with the migration ID.